### PR TITLE
Cat 1248 initial cat update the 2nd time around decision page so that it works for womens estate

### PR DESCRIPTION
**Pull request by Olusola-Opencast**
CAT-1248 update initial category decision page second time round for womens estate.

### CODE REVIEW

File: server/views/formPages/openConditions/provisionalCategory.html [hunk 1]
```diff
@@ -25,14 +25,23 @@
 
 <h1 class="govuk-heading-l">{{ pageTitle }}</h1>
 
+{% set isFemale = user.activeCaseLoad.female %}
+
 <div class="govuk-warning-text forms-info-text warning-overrides">
-  <span class="govuk-warning-text__icon" aria-hidden="true">{{ cat }}</span>
+  {% if (not isFemale and cat == 'D') or isFemale %}
```
Comment (kberec) [inline server/views/formPages/openConditions/provisionalCategory.html:31]: I think this should be always exclamation mark

File: server/views/formPages/openConditions/provisionalCategory.html [hunk 2]
```diff
@@ -25,14 +25,23 @@
...
+    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  {% else %}
+    <span class="govuk-warning-text__icon" aria-hidden="true">{{ cat }}</span>
+  {% endif %}
   <strong class="govuk-warning-text__text">
     <span class="govuk-warning-text__assistive">Warning</span>
-    Based on the information provided, the provisional category is {{ catDisplay(cat) }}
+    {% if cat == 'D' %}
```
Comment (kberec) [inline server/views/formPages/openConditions/provisionalCategory.html:38]: I think this should be `{% if cat == 'D' or cat == 'T' or cat == 'J' %}`

File: server/views/formPages/openConditions/provisionalCategory.html [hunk 3]
```diff
@@ -25,14 +25,23 @@
...
+      The provisional category is open
+    {% else %}
+      The provisional category is {{ catDisplay(cat) }}
```
Comment (kberec) [inline server/views/formPages/openConditions/provisionalCategory.html:41]: I think this should be `The provisional category is closed`

Comment (Olusola-Opencast) [reply server/views/formPages/openConditions/provisionalCategory.html:41]: I have checked this with Anthony:

For mens category D and womens category T the warning is: "! The provisional category is open"
For mens category B the warning is: "B The provisional category is category B
For mens category C the warning is: "C The provisional is category C"
For womens category R the warning is: "! The provisional category is closed"

**Pull request merged**